packages/api/src/controllers/v1/index.ts

```diff
@@ -70,15 +70,15 @@ export class V1 {
 			contact = await prisma.contact.create({
 				data: {
 					email,
-					subscribed: subscribed ?? true,
```

Comment:
This is breaking behaviour. If a contact is first created, it should be put on subscribed unless explicitly mentioned otherwise